.env.example

```
@@ -37,7 +37,13 @@ ELASTICSEARCH_MAX_RETRIES=3
 # Optional: Bearer token for MCP server authentication
 AUTH_TOKEN=
 
-# Server settings
+# Server settings (used by run.sh script)
+MCP_HOST=localhost
+MCP_PORT=8005
+MCP_TRANSPORT=http
+MCP_LOG_FILE=logs/syslog_mcp_server.log
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The MCP_LOG_FILE key should go before the MCP_PORT key

(UnorderedKey)